include/revng/TupleTree/Visits.h

```diff
@@ -271,6 +272,10 @@ bool callOnPathSteps(Visitor &V,
   return tupletree::detail::callOnPathStepsTuple(V, Path, M, FullPath);
 }
 
+template<typename T>
+concept IsTrackingContainer = StrictSpecializationOf<T,
+                                                     revng::TrackingContainer>;
```

Comment:
I'd check if it has `tryGet`.

Reply:
i considered it, but i think this is more correct, bucause the issue arises not from people using vectors that do not have try get, but from the fact that one may generate a model like from the yaml without instrumentation, so here we actually want to differentiate those to cases.